imports/api/users/server/publications.js

```diff
@@ -12,6 +12,10 @@ Meteor.publish('users', function() {
   return Meteor.users.find();
 });
 
+Meteor.publish('profiles', function() {
+  return Profiles.find();
+});
```

Comment:
I think we've talked about subscriptions like this. We need to limit them only to provide the minimal set of information required. Especially important in this case where lots of semi-personal information is involved. Just set this to return a single result, cool?